.gitignore

```diff
@@ -1,7 +1,10 @@
+data
```

Comment:
Yes. Simplified it a bit now by removing the other `data/...` entries. We have to rely on git's ability to add files to the repo that are under .gitignore-d directories.